additionalCSS/footerSection.css

```diff
@@ -15,13 +15,15 @@
 .click:hover {
   cursor: pointer;
   background-color: #008552;
+  transform: scale(1.1);
 }
 
 .contactForm {
   display: flex;
   flex-direction: column;
   justify-content: center;
-  padding: 0 24px;
+  width: 90%;
+  margin: 0 auto;
```

Comment:
- Kindly align the contact desc and form based on the above cards to stick to the Figma design and have a better UI 👌
### Your design 👇
![2023-05-05_09-14](https://user-images.githubusercontent.com/52010789/236389552-97424a83-edeb-4d69-a616-d784f48c38ca.png)
### Figma design 👇
![2023-05-05_09-16](https://user-images.githubusercontent.com/52010789/236389562-8231ee49-73de-457f-b401-c330a92ea393.png)